.github/workflows/assets.yaml

```diff
@@ -110,8 +144,10 @@ jobs:
         kubectl wait --for=condition=available --timeout=60s deploy/httpbin
     - name: iter8 k launch
       run: |
-        iter8 k launch -c load-test-http \
-        --set url="http://httpbin.default/get"
+        iter8 k launch \
+        --set tasks={http} \
```

Comment:
Notice how `--set tasks={http}` replaces the `--chartName load-test-http` mode of creating experiments.
***
There is only one experiment chart now, helpfully named `iter8`, and no longer required to be specified by the user. You can flexibly compose an experiment by simplify specifying the tasks... 

For e.g., `--set tasks={http,assess,notify}` would put together an experiment with these three tasks (`notify` is yet-to-be created).